### PR TITLE
thumbnail cleanup + gif

### PR DESCRIPTION
**Pull request by lindsayplatt**
Making thumbnail have grey outlines + cleaning up some code + a GIF (#35)

### CODE REVIEW

File: scripts/visualize/visualize_utils.R [hunk 1]
```diff
@@ -209,40 +214,26 @@ locate_css_class_detail <- function(css, item_nm){
   return(item_details)
 }
 
-#' script to turn the dataviz into a thumbnail
-#' 
-visualize.map_thumbnail <- function(viz){
+createHurricaneSnapshot <- function(fig_height, fig_width, css, time_stamp, states, islands,
```
Comment (dblodgett-usgs) [inline scripts/visualize/visualize_utils.R]: Nice refactor into functions. Would be helpful to add just a little documentation of the purpose of these for later readability.

File: scripts/visualize/visualize_utils.R [hunk 2]
```diff
@@ -251,30 +242,81 @@ visualize.map_thumbnail <- function(viz){
...
-  sp::plot(counties, add = TRUE, col = counties@data$col)
+  sp::plot(counties, col = NA, border = "#c6c6c6",
+           expandBB = eval(parse(text = expandBB))) #decimals not working?
+  sp::plot(states, col = state_color, border = "#c6c6c6", add = TRUE)
+  sp::plot(islands, add = TRUE, border = "#c6c6c6", col = island_color)
+  sp::plot(counties, add = TRUE, border = "#c6c6c6", col = counties@data$col)
   sp::plot(hurricane_track, add=TRUE, col = "black", lwd=3)
   cols <- rep("#FFFFFF00", length(storm))
-  cols[time.idx] <- 'red'
+  cols[time_idx] <- storm_color
```
Comment (dblodgett-usgs) [inline scripts/visualize/visualize_utils.R]: The never ending battle ... `.` or `_` !

**Comment by dblodgett-usgs** [comment]
👍  Looks good to me!

File: scripts/visualize/visualize_utils.R (hunk 2 above)
Comment (lindsayplatt) [reply scripts/visualize/visualize_utils.R]: lol ... yeah, just trying to be consistent within the function

File: scripts/visualize/visualize_utils.R (hunk 1 above)
Comment (lindsayplatt) [reply scripts/visualize/visualize_utils.R]: Good point! I will work on that

**Comment by lindsayplatt** [comment]
current state:

image

**Pull request merged**